src/core/algorithm/graphToMarkovChain.test.js

```diff
@@ -203,6 +228,33 @@ describe("core/algorithm/graphToMarkovChain", () => {
       expect(normalize(osmc)).toEqual(normalize(expected));
     });
 
+    it("works on a simple chain with a loop edge", () => {
+      // The loop edge will implicitly correspond to one forward edge with weight 2, and one backward edge with weight 1
```

Comment:
Why is this "implicit"? Doesn't this match the explicit weight resolver?
`const edgeWeight = () => ({forwards: 2, backwards: 1});`